pydoc/user_guide/concepts.md

```diff
@@ -0,0 +1,188 @@
+Relation to SOFT
```

Comment:
There is already an updated doc/concepts.md file

Reply:
Sure. This is a verbatim copy of the same file under `doc/`.